crypto/fipsmodule/evp/p_hmac.c

```diff
@@ -81,6 +81,10 @@ static int hmac_copy(EVP_PKEY_CTX *dst, EVP_PKEY_CTX *src) {
   sctx = src->data;
   dctx = dst->data;
   dctx->md = sctx->md;
+  if(sctx->ktmp.key != NULL && !HMAC_KEY_copy(&sctx->ktmp, &dctx->ktmp)) {
```

Comment:
Do we need to guard `sctx->ktmp` before accessing the internals?
```suggestion
  if(sctx->ktmp != NULL && sctx->ktmp.key != NULL && !HMAC_KEY_copy(&sctx->ktmp, &dctx->ktmp)) {
```

Reply:
No `sctx->ktmp` is a struct, not a pointer to a struct.